lnrpc/neutrinorpc/neutrino.proto

```diff
@@ -244,3 +249,15 @@ message GetBlockHashResponse {
     // The block hash.
     string hash = 1;
 }
+
+message UnbanPeerRequest {
+    // The address of the peer to be unbanned.
+    string peer_address = 1;
+
+    // Indicates if the peer to be unbanned should be connected as a
+    // permanent peer.
+    bool permanent = 2;
+}
+
+message UnbanPeerResponse {
+}
```

Comment:
i think the general pattern we are aiming towards with responses is to include a `string status` so that the user sees some sort of success message instead of just an empty JSON struct. 

See `message BumpFeeResponse {` for an example of this